src/ResourceManager/Resources/Commands.Resources/Models.Authorization/AuthorizationClient.cs

```diff
@@ -663,20 +594,9 @@ private PSRoleDefinition CreateOrUpdateRoleDefinition(Guid roleDefinitionId, PSR
                     Type = "CustomRole"
                 }
             };
-#endif
-            try
-            {
-                roleDef = AuthorizationManagementClient.RoleDefinitions.CreateOrUpdate(
-#if !NETSTANDARD
-					roleDefinitionId, roleDefinition.AssignableScopes.First(), 
-#else
-					roleDefinition.AssignableScopes.First(), roleDefinitionId.ToString(),
-#endif
-					parameters)
-#if !NETSTANDARD
-					.RoleDefinition
```

Comment:
Yes, because when we use '!NETSTANDARD' Nuget generated from azure-sdk-for-net, we don't need to use '.RoleDefinition ', So I deleted it.